.github/workflows/main-build.yml

```diff
@@ -5,8 +5,15 @@ on:
     branches: 
       - main 
       - release/*
+    paths-ignore:
+      - '.github/workflows/main-build.yml'
+      - '**.md'
```

Comment:
This still ignores when changes are made to the workflow file. Your initial commit was only missing an extra line. You could refer to the [Collector workflow here](https://github.com/aws-observability/aws-otel-collector/blob/0db5cf8136d8ca06efcc9be47124b4d7bb55affa/.github/workflows/CI.yml#L22) as an example.

Reply:
oh i see your point now, modified.